.fixtures.yml

```diff
@@ -1,7 +1,8 @@
 fixtures:
-  repositories:
-    stdlib: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
-    apt: "https://github.com/puppetlabs/puppetlabs-apt.git"
-    vcsrepo: "https://github.com/puppetlabs/puppetlabs-vcsrepo.git"
```

Comment:
hmm why are we switching these? It seems like puppetlabs modules still use the repos.
https://github.com/puppetlabs/puppetlabs-apache/blob/master/.fixtures.yml

I also like testing against newer code, so we can be aware sooner of upcoming changes.

Reply:
Oh, that's true! It's for testing, then it makes sense. (I thought I would do you a favor, sorry!) :flushed: